ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json

```diff
@@ -166,5 +166,19 @@
     "CaseEventID": "populateCase-PREPARE_FOR_HEARING",
     "UserRole": "caseworker-publiclaw-solicitor",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "returnApplication",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "returnApplication",
+    "UserRole": "[LASOLICITOR]",
+    "CRUD": "R"
```

Comment:
Left in nonprod as this story is awaiting a story to be started / complete to introduce notifiactions.